index.bs

```diff
@@ -776,19 +789,19 @@ interface XRReferenceSpace : XRSpace {
 };
 </pre>
 
-Each {{XRReferenceSpace}} has a <dfn for="XRReferenceSpace">type</dfn>, which is an {{XRReferenceSpaceType}}.
+The [=XRSpace/internal type=] of an {{XRReferenceSpace}} is one of the variants of {{XRReferenceSpaceType}}, chosen during construction..
 
 An {{XRReferenceSpace}} is most frequently obtained by calling {{XRSession/requestReferenceSpace()}}, which creates an instance of an {{XRReferenceSpace}} or an interface extending it, determined by the {{XRReferenceSpaceType}} enum value passed into the call. The type indicates the tracking behavior that the reference space will exhibit:
 
-  - Passing a type of <dfn enum-value for="XRReferenceSpaceType">identity</dfn> creates an {{XRReferenceSpace}} instance. It represents a reference space where the [=viewer=] is always at the [=native origin=]. Only the {{XRSession/viewerSpace}} and {{XRInputSource/targetRaySpace}}'s of {{XRInputSource}}'s with an {{XRInputSource/targetRayMode}} of {{XRTargetRayMode/screen}} can be spatially related to an {{identity}} reference space.
+  - Passing a type of <dfn enum-value for="XRReferenceSpaceType">identity</dfn> creates an {{XRReferenceSpace}} instance. It represents a reference space where the [=viewer=] is always at the [=native origin=]. Only the {{XRSession/viewerSpace}} and {{XRInputSource/targetRaySpace}}'s of {{XRInputSource}}'s with an {{XRInputSource/targetRayMode}} of {{XRTargetRayMode/screen}} can be spatially related to an {{identity}} reference space. Its [=native origin=] is the position and orientation of the [=viewer=].
 
-  - Passing a type of <dfn enum-value for="XRReferenceSpaceType">position-disabled</dfn> creates an {{XRReferenceSpace}} instance. It represents a tracking space where orientation is tracked but the [=viewer=]s position is always at the [=native origin=].
+  - Passing a type of <dfn enum-value for="XRReferenceSpaceType">position-disabled</dfn> creates an {{XRReferenceSpace}} instance. It represents a tracking space where orientation is tracked but the [=viewer=]s position is always at the [=native origin=]. Its [=native origin=] has a position matching the position of the [=viewer=], and a constant origin initialized based on the conventions of the underlying [=/XR device=].
```

Comment:
> Its [=native origin=] has a position matching the position of the [=viewer=]

This feels like a repetition of the line above it. I'm fine with keeping your newer verbiage, but we should condense it down to one line.

> and a constant origin initialized based on the conventions of the underlying [=/XR device=]

Is this meant to say "and a constant **orientation**"?

Reply:
Whoops.

Same situation as the identity one. I can clarify that and instead mention its behavior with getViewerPose specifically to inform API users what it's for.

Reply:
Done. Left an additional non-normative line about the kind of experiences this is for.